src/index.ts

```diff
@@ -40,6 +40,17 @@ export async function main(): Promise<void> {
       ),
     );
 
+    // Assumption: The filepath is of the format `teams/{team-name}` or `teams/{team-name}/migrations` (for db migrations)
+    const team =
```

Comment:
We should move this to `update-promoted-values`, since `promotion-target-regexip` will be null for things other things.